### DISCUSSION TITLE
can we subscribe to multiple records in one collection

### DISCUSSION THREAD

**Discussion by nikhil-varma-mudunuri** [pocketbase/pocketbase#4021] [Q&A] [answered]
example : 
i have a collection with some 400 records, out of which 4 belongs to specific user, can i subscribe to those 4 records  , can i filter specific records and subscribe to those records

**Comment by ganigeorgiev** [comment, marked as answer]
You can subscribe to the wildcard topic (`*`) or to a single record.

The realtime connected client will receive events **only** for the records that satisfy the List (for the wildcard) or View API (for the single record) rules of the collection.

In addition to the topic, with PocketBase v0.20 (_requires also JS SDK v0.20+_) the realtime subscriptions now also support the `filter` query param if you want to further filter the received events. For example:

```js
pb.collection("example").subscribe("*", (e) => {
    console.log(e.record);
}, {
    filter: "someField > 10"
})
```
